src/main/java/com/minecolonies/coremod/colony/buildings/modules/settings/BuilderModeSetting.java

```diff
@@ -0,0 +1,41 @@
+package com.minecolonies.coremod.colony.buildings.modules.settings;
+
+import com.ldtteam.structurize.Structurize;
+import com.ldtteam.structurize.placement.StructureIterators;
+import com.minecolonies.api.colony.buildings.modules.settings.ISettingsModuleView;
+
+import java.util.List;
+
+import static com.minecolonies.api.research.util.ResearchConstants.BUILDER_MODE;
+
+/**
+ * Stores the builder mode setting.
+ */
+public class BuilderModeSetting extends StringSetting
+{
+    /**
+     * Create the builder mode setting.
+     */
+    public BuilderModeSetting()
+    {
+        super(StructureIterators.getKeySet(), 0);
+        set(Structurize.getConfig().getServer().iteratorType.get());
```

Comment:
There's a high probability that this will only read the default value from the Structurize config, not the currently-set value in a custom config.